lib/Command/Updater/Job.php

```diff
@@ -64,12 +73,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $output->writeln("Checking update Status");
         $date = new DateTime();
         $date->setTimestamp($this->statusService->getUpdateTime());
-        $output->writeln("Last Execution was ".$date->format('Y-m-d H:i:s e'));
+        $now = new DateTime('now');
+        $elapsedInterval = $now->diff($date);
+        $output->writeln("Last Execution was ".$date->format('Y-m-d H:i:s e').
+          $elapsedInterval->format("; %h hours, %i minutes, %s seconds ago"));
 
         if ($reset) {
             $output->writeln("Attempting to reset the job.");
             $this->updaterService->reset();
             $output->writeln("Done, job should execute on next schedule.");
+        } else {
```

Comment:
My thinking was that when I'm performing a deliberate reset, that would be the action I take when I already know something's wrong. 

In addition, if the reset is successful, it would be confusing for me to be told "something's wrong", with an error code.

Reply:
Sure, so we can return at the end of the reset. But we shouldn't hook up the error to the same logic.